contracts/libraries/SafeMigration.sol

```diff
@@ -77,9 +77,9 @@ contract SafeMigration is SafeStorage {
      * @notice Migrate to Safe Singleton and set the fallback handler. This function is intended to be used when migrating
      *         a Safe to a version which also requires updating fallback handler.
      */
-    function migrateWithFallbackHandler() public onlyDelegateCall {
+    function migrateWithFallbackHandler() external onlyDelegateCall {
         migrateSingleton();
-        ISafe(address(this)).setFallbackHandler(SAFE_FALLBACK_HANDLER);
+        ISafe(payable(address(this))).setFallbackHandler(SAFE_FALLBACK_HANDLER);
```

Comment:
Is `payable` needed here? Oddly, it isn't needed in other places like:

https://github.com/safe-global/safe-smart-account/blob/44aa3acba0f3a7b9f129037aa468c0fdb118c57e/contracts/libraries/SafeToL2Migration.sol#L105

Reply:
It looks like it isn't needed. Will make the change here as well as backport that change to this branch `release/v1.4.1-2`

Reply:
Done 👍🏾